mne/viz/_3d.py

```diff
@@ -666,8 +856,8 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
 
     for key in surfs.keys():
         surfs[key] = transform_surface_to(surfs[key], coord_frame, mri_trans)
-    src_rr = apply_trans(mri_trans, src_rr)
-    src_nn = apply_trans(mri_trans, src_nn, move=False)
+    # src_rr = apply_trans(mri_trans, src_rr)
+    # src_nn = apply_trans(mri_trans, src_nn, move=False)
```

Comment:
I agree it looks wrong but I don't see why it should be. The `src` should be in MRI coords, and the `mri_trans` should get applied to all points in the MRI coordinate frame, right? Or do we already transform `src` to the destination coordinate frame elsewhere in the code?